tests/test_dvhcalc.py

```diff
@@ -259,6 +262,316 @@ def test_dvh_calculation_with_interpolation_between_planes(self):
         self.assertAlmostEqual(dvh.mean, 6.4767105)
 
 
+class TestDVHCalcDecubitus(unittest.TestCase):
+    """Unit tests for DVH calculation in decubitus orientations."""
+
+    def setUp(self):
+        """Set up fake DICOM datasets used in various tests."""
+        self.ss = fake_ss()
+        self.dose = fake_rtdose()
+
+    def test_nondecub(self):
+        """Test that DVH is calculated correctly for standard orientation."""
+        self.dose.ImageOrientationPatient = [1, 0, 0, 0, 1, 0]
+        dvh = get_dvh(self.ss, self.dose, 1)
+        diffl = dvh.differential
+        # Counts are normalized to total, and to volume,
+        # So undo that here for test dose grid.
+        # 18=num dose voxels inside struct; 0.36=volume
+        got_counts = diffl.counts * 18 / 0.36
+        expected_counts = [0]*13 + [2, 2, 2, 0, 0, 0, 0, 0, 0, 0,
+                                    2, 2, 2, 0, 0, 0, 0, 0, 0, 0, 2, 2, 2]
+        assert numpy.all(numpy.isclose(got_counts, expected_counts))
```

Comment:
We have used `numpy` testing functions such as `numpy.testing.assert_allclose` in the other test suites, i.e. in`test_dicomparser`. I thought it might be good to use those here as well in `TestDVHCalcDecubitus`. I changed it in my checkout, so I can push the edits to `test_dvhcalc.py` to your branch. Let me know what you think.

Reply:
Sure, I just wasn't aware of that numpy testing function.

Reply:
Resolved in f87d514.